packages/core/src/loadBalancer/CreateLoadBalancerButton.tsx

```diff
@@ -38,59 +35,11 @@ export class CreateLoadBalancerButton extends React.Component<ICreateLoadBalance
     });
   }
 
-  private createLoadBalancerProviderFilterFn = (
-    _app: Application,
-    _acc: IAccountDetails,
-    provider: ICloudProviderConfig,
-  ): boolean => {
-    const lbConfig = provider.loadBalancer;
-    return (
-      lbConfig &&
-      (lbConfig.CreateLoadBalancerModal ||
-        (lbConfig.createLoadBalancerTemplateUrl && lbConfig.createLoadBalancerController))
-    );
-  };
-
-  private createLoadBalancer = (): void => {
-    const { app } = this.props;
-    ProviderSelectionService.selectProvider(app, 'loadBalancer', this.createLoadBalancerProviderFilterFn).then(
-      (selectedProvider) => {
-        const provider = CloudProviderRegistry.getValue(selectedProvider, 'loadBalancer');
-
-        if (provider.CreateLoadBalancerModal) {
-          provider.CreateLoadBalancerModal.show({
-            app: app,
-            application: app,
-            forPipelineConfig: false,
-            loadBalancer: null,
-            isNew: true,
-          });
-        } else {
-          // angular
-          ModalInjector.modalService
-            .open({
-              templateUrl: provider.createLoadBalancerTemplateUrl,
-              controller: `${provider.createLoadBalancerController} as ctrl`,
-              size: 'lg',
-              windowClass: 'modal-z-index',
-              resolve: {
-                application: () => this.props.app,
-                loadBalancer: (): ILoadBalancer => null,
-                isNew: () => true,
-                forPipelineConfig: () => false,
-              },
-            })
-            .result.catch(() => {});
-        }
-      },
-    );
-  };
-
   public render() {
     if (!this.state.isDisabled) {
       return (
         <div>
-          <button className="btn btn-sm btn-default" onClick={this.createLoadBalancer}>
+          <button className="btn btn-sm btn-default" disabled={true}>
```

Comment:
btw for booleans you only need to have the attribute present `disabled` (`={true}` isn't needed).